app/src/pages/embedding/ExportSelectionButton.tsx

```diff
@@ -88,31 +127,27 @@ export function ExportSelectionButton() {
                   Download
                 </Button>
               }
+            ></Alert>
+            <div
+              css={css`
+                display: flex;
+                flex-direction: column;
+                justify-content: space-between;
+                align-items: flex-start;
+                padding: 16px;
+                gap: 8px;
+              `}
             >
-              <div
+              <p
                 css={css`
-                  display: flex;
-                  flex-direction: row;
-                  justify-content: space-between;
-                  align-items: center;
-                  button {
-                    flex: none;
-                  }
+                  margin: 0;
+                  flex: 1 1 auto;
                 `}
               >
-                <p
-                  css={css`
-                    margin: 0;
-                    flex: 1 1 auto;
-                  `}
-                >
-                  <span>
-                    You can retrieve your export in your notebook via{" "}
-                  </span>
-                  <b>px.active_session().exports</b>
-                </p>
-              </div>
-            </Alert>
+                <span>You can retrieve your export in your notebook via </span>
+              </p>
+              <CodeBlock value={EXPORTS_CODE_SNIPPET} />
```

Comment:
```suggestion
              <View borderColor="light" ... >
              <CodeBlock value={EXPORTS_CODE_SNIPPET} />
              </View>
```
I would add a border and some rounding here to contain the content nicely

Reply:
addressed in a1dd8447